### ISSUE TITLE
Tactile force feild image training failed

### ISSUE THREAD

**Issue by IreneLu12** [NVIDIA-Omniverse/IsaacGymEnvs#255]
I am using the following training script for Reduced State + Tactile-FF training, but the reward is very low, and the success rate remains zero. Could it be due to incorrect configuration settings? What should the correct configuration be?

```
python train.py \
    'task=TacSLTaskInsertion' 'task.env.task_type=insertion' \
    'seed=-1' 'task.env.use_gelsight=True' 'headless=True' \
    'max_iterations=1500' 'task.env.numEnvs=96' \
    'task.randomize.plug_noise_rot_in_gripper=[0.0, 0.628318, 0.0]' \
    'train=TacSLTaskInsertionPPO_LSTM_dict_AAC' 'train.params.config.horizon_length=512' \
    'train.params.config.mini_epochs=4' \
    +'task.env.obsDims={ee_pos:[3],ee_quat:[4],socket_pos:[3],socket_quat:[4],dof_pos:[9]}' \
    +'train.params.network.input_preprocessors={ee_pos:{},ee_quat:{},socket_pos:{},socket_quat:{},dof_pos:{}}' \
    'task.rl.asymmetric_observations=True' 'task.rl.add_contact_info_to_aac_states=True' \
    +'task.env.stateDims={ee_pos:[3],ee_quat:[4],plug_pos:[3],plug_quat:[4],socket_pos_gt:[3],socket_quat:[4],dof_pos:[9],ee_lin_vel:[3],ee_ang_vel:[3],plug_socket_force:[3],plug_left_elastomer_force:[3],plug_right_elastomer_force:[3]}' \
    +'train.params.config.central_value_config.network.input_preprocessors={ee_pos:{},ee_quat:{},plug_pos:{},plug_quat:{},socket_pos_gt:{},socket_quat:{},dof_pos:{},ee_lin_vel:{},ee_ang_vel:{},plug_socket_force:{},plug_left_elastomer_force:{},plug_right_elastomer_force:{}}' \
    'task.rl.add_contact_force_plug_decomposed=True' \
    'task.env.use_shear_force=True' 'task.env.use_tactile_field_obs=True' 'task.env.zero_out_normal_force_field_obs=True' 'task.env.use_camera=False'  \
    +'task.env.obsDims={tactile_force_field_left:[30,20,3], tactile_force_field_right:[30,20,3]}' \
    experiment=insert_tactile_withff
```

**Comment by BR-0462** [comment]
> 我正在使用以下训练脚本，用于减少状态+触觉-FF训练，但奖励非常低，成功率依然为零。会不会是配置设置不正确导致的？正确的配置应该是什么？
> 
> ```
> python train.py \
>     'task=TacSLTaskInsertion' 'task.env.task_type=insertion' \
>     'seed=-1' 'task.env.use_gelsight=True' 'headless=True' \
>     'max_iterations=1500' 'task.env.numEnvs=96' \
>     'task.randomize.plug_noise_rot_in_gripper=[0.0, 0.628318, 0.0]' \
>     'train=TacSLTaskInsertionPPO_LSTM_dict_AAC' 'train.params.config.horizon_length=512' \
>     'train.params.config.mini_epochs=4' \
>     +'task.env.obsDims={ee_pos:[3],ee_quat:[4],socket_pos:[3],socket_quat:[4],dof_pos:[9]}' \
>     +'train.params.network.input_preprocessors={ee_pos:{},ee_quat:{},socket_pos:{},socket_quat:{},dof_pos:{}}' \
>     'task.rl.asymmetric_observations=True' 'task.rl.add_contact_info_to_aac_states=True' \
>     +'task.env.stateDims={ee_pos:[3],ee_quat:[4],plug_pos:[3],plug_quat:[4],socket_pos_gt:[3],socket_quat:[4],dof_pos:[9],ee_lin_vel:[3],ee_ang_vel:[3],plug_socket_force:[3],plug_left_elastomer_force:[3],plug_right_elastomer_force:[3]}' \
>     +'train.params.config.central_value_config.network.input_preprocessors={ee_pos:{},ee_quat:{},plug_pos:{},plug_quat:{},socket_pos_gt:{},socket_quat:{},dof_pos:{},ee_lin_vel:{},ee_ang_vel:{},plug_socket_force:{},plug_left_elastomer_force:{},plug_right_elastomer_force:{}}' \
>     'task.rl.add_contact_force_plug_decomposed=True' \
>     'task.env.use_shear_force=True' 'task.env.use_tactile_field_obs=True' 'task.env.zero_out_normal_force_field_obs=True' 'task.env.use_camera=False'  \
>     +'task.env.obsDims={tactile_force_field_left:[30,20,3], tactile_force_field_right:[30,20,3]}' \
>     experiment=insert_tactile_withff
> ```

Hello, did you configure your environment according to the official documentation and successfully run `./train_state.sh`? I'm having trouble running it. Could you please tell me your PyTorch and rl_games versions?
